index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <div class="container">
+        <header class="app-bar">
+            <div class="status-bar"><img alt="status bar" src="images/statusbar.png"></div>
+            <nav>
+                <a href="index.html"><h2>Jonas</h2></a>
+                <div>
+                    <a><img alt="overflow-image" src="images/icon_menu_img.png"></a>
+                </div>
+            </nav>
+        </header>
+
+        <section class="header-section">
+            <h1 class="title">I'm Jonas <br>Glad to see you</h1>
```

Comment:
- [ Optional ] Good Job for stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly adjust the alignment of the title just like the figma design. 

Your Project Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/206187095-936a2bcf-ac95-4af8-a937-4005b904e3fd.PNG)

The Figma design Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/206187578-e7d1c4df-06bb-418a-9ee4-8306b77943ac.PNG)

Reply:
Thank you. Will correct it right away